code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -48,7 +48,7 @@ obj/item/gun/energy/retro
 	name = "antique laser gun"
 	icon_state = "caplaser"
 	item_state = "caplaser"
-	desc = "This is an antique laser gun. All craftsmanship is of the highest quality. It is decorated with assistant leather and chrome. The object menaces with spikes of energy. On the item is an image of Baystation 12. The station is exploding."
+	desc = "This is an antique laser gun. Its craftsmanship is of the highest quality. The object menaces with spikes of energy."
```

Comment:
The all should remain, instead of its. It is a dwarf fortress reference, even if half of it is gone.

Reply:
Oh, I thought that was just a typo. Will fix